discordbot/economy/currencies.py

```diff
@@ -5,34 +5,61 @@
 
 
 async def currencies_command(ctx, arg):
-    # Help
-    if arg == "-h" or arg == "help":
-        help_txt = "Currencies [Source: Wall St. Journal]\n"
-        embed = discord.Embed(
-            title="Economy: [WSJ] Currencies HELP",
-            description=help_txt,
-            colour=cfg.COLOR,
-        )
-        embed.set_author(
-            name=cfg.AUTHOR_NAME,
-            icon_url=cfg.AUTHOR_ICON_URL,
-        )
+    try:
+        # Debug
+        if cfg.DEBUG:
+            print("-- STARTED COMMAND: !stocks.economy.currencies " + arg + " --")
+
+        # Help
+        if arg == "-h" or arg == "help":
+            help_txt = "Currencies [Source: Wall St. Journal]\n"
+            embed = discord.Embed(
+                title="Economy: [WSJ] Currencies HELP",
+                description=help_txt,
+                colour=cfg.COLOR,
+            )
+            embed.set_author(
+                name=cfg.AUTHOR_NAME,
+                icon_url=cfg.AUTHOR_ICON_URL,
+            )
 
-        await ctx.send(embed=embed)
+            await ctx.send(embed=embed)
 
-    else:
-        df_data = wsj_model.global_currencies()
-        if df_data.empty:
-            df_data_str = "No currencies data available"
         else:
-            df_data_str = "```" + df_data.to_string(index=False) + "```"
+            df_data = wsj_model.global_currencies()
+            if df_data.empty:
+                df_data_str = "No currencies data available"
+            else:
+                df_data_str = "```" + df_data.to_string(index=False) + "```"
 
-        embed = discord.Embed(
-            title="Economy: [WSJ] Currencies", description=df_data_str, colour=cfg.COLOR
-        )
+            embed = discord.Embed(
+                title="Economy: [WSJ] Currencies",
+                description=df_data_str,
+                colour=cfg.COLOR,
+            )
+            embed.set_author(
+                name=cfg.AUTHOR_NAME,
+                icon_url=cfg.AUTHOR_ICON_URL,
+            )
+
+            await ctx.send(embed=embed)
+
+    except Exception as e:
+        title = "INTERNAL ERROR"
+        embed = discord.Embed(title=title, colour=cfg.COLOR)
         embed.set_author(
             name=cfg.AUTHOR_NAME,
             icon_url=cfg.AUTHOR_ICON_URL,
         )
-
-        await ctx.send(embed=embed)
+        embed.set_description(
+            "Try updating the bot, make sure DEBUG is True in the config "
+            "and restart it.\nIf the error still occurs open a issue at: "
+            "https://github.com/GamestonkTerminal/GamestonkTerminal/issues"
+        )

```

Comment:
If we want the user to see this we need a `await ctx.send(embed=embed)` I think

Reply:
Since the error will be the same with the exception with what is caught by the exception, should we have this default embed in config in a function, something like:
```
def exception_embed(e: Exception):
        embed = discord.Embed(title="EXCEPTION", colour=cfg.COLOR)
        embed.set_author(
            name=cfg.AUTHOR_NAME,
            icon_url=cfg.AUTHOR_ICON_URL,
        )
        embed.set_description(e)
```

Reply:
I forgot the ```await ctx.send(embed=embed)``` part.